packages/notification-center/src/i18n/languages/mr.ts

```diff
@@ -0,0 +1,11 @@
+import { ITranslationEntry } from '../lang';
+
+export const HR: ITranslationEntry = {
+  translations: {
+    notifications: 'सुचना',
+    markAllAsRead: 'सर्व वाचल्याप्रमाणे चिन्हांकित करा',
+    poweredBy: 'द्वारा समर्थित',
+    settings: 'सेटिंग्स',
```

Comment:
Google Translate gives me some inconsistencies regarding capitalisation, could you review them please?